SparkleSpin/Views/EntryCell/EntryCell.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  EntryCell.swift
+//  SparkleSpin
+//
+//  Created by Britney Smith on 3/17/19.
+//  Copyright © 2019 Britney Smith. All rights reserved.
+//
+
+import UIKit
+
+class EntryCell: UITableViewCell {
```

Comment:
Are you thinking in implementing something later in this cell?

Reply:
Hmm, I used this class for something before in previous iterations of what I was trying to do. I _may_ need it for other logic that I need to implement for the full functionality though. If I don't end up using it, I'll delete it in the next PR.